config/offices.yml

```diff
@@ -0,0 +1,27 @@
+# APP_RELEASE_STAGE
+defaults: &defaults
+  # Which office to send the fax to in the event the submitted
+  # zip code is not in our mapping
+  fallback_office: "union"
+
+  # Because YML inheritance doesn't work well with nested hashes,
+  # We are relying on matching the list of offices per-environment with
+  # the list of zip codes a given office serves to reduce configuration duplication.
+  # This may be unnecessary?
+  covered_zip_codes:
+    clio: [48415,48417,48420,48421,48433,48437,48457,48458,48460,48463,48464,48473,48504,48506,48531,48532,48746]
```

Comment:
I think that our rubocop config pefers spaces after commas within arrays. Or is not doing that a yml thing?

Reply:
Oooh good point. I was doing regexs to transform this and lost the space.